src/components/CreatePostInput/index.js

```diff
@@ -214,14 +199,9 @@
         handleStateHashtag={handleStateHashtag}
         hashtagPosition={hashtagPosition}
         positionTopicSearch={positionTopicSearch}
-        // setHashtags={setHashtags}
         setPositionKeyboard={setPositionKeyboard}
         setMessage={setMessage}
-        setTopicChats={setTopicChats}
         setTopicSearch={setTopicSearch}
-        setTopics={setTopics}
-        topicChats={topicChats}
-        topics={topics}
         topicSearch={topicSearch}
       />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several props have been removed from this component. Make sure that these changes don't affect the expected behavior of the child components that rely on these props. If these props are no longer needed, you can disregard this comment.
```diff
-        setTopicChats={setTopicChats}
-        setTopics={setTopics}
-        topicChats={topicChats}
-        topics={topics}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->